mftf/2.3/test/action-groups.md

```diff
@@ -39,17 +39,19 @@ The XML format for the `actionGroups` declaration is:
 ```
 
 ## Example
-{%raw%}
+
+{% raw %}
 
 These examples build a declaration for a group of actions that grant authorization to the Admin area, and use the declaration in a test.
 
-The _Backend/ActionGroup/LoginToAdminActionGroup.xml_ `<actionGroup>` relates to the functionality of the _Backend_ module. In [test](../test.html), the name and identifier of the `<actionGroup>` is used as a reference in the `ref` parameter, such as `ref="LoginToAdminActionGroup"`.
+The _Backend/ActionGroup/LoginToAdminActionGroup.xml_ `<actionGroup>` relates to the functionality of the _Backend_ module.
+In [test](../test.html), the name and identifier of the `<actionGroup>` is used as a reference in the `ref` parameter, such as `ref="LoginToAdminActionGroup"`.
```

Comment:
Are relative paths (../test.html) OK? I've been admonished for doing so.

Reply:
It is a must for multi source projects.